scripts/Install-VSCode.ps1

```diff
@@ -133,118 +133,402 @@ param(
     [ValidateSet("Stable", "Insider-System", "Insider-User")]
     [string]$BuildEdition = "Stable",
 
-    [Parameter()]
-    [switch]$User,
```

Comment:
This parameter didn't seem to be used...

Reply:
Hmm sure it wasn't used for User vs System install?

Reply:
Yeah I had a search through. `$BuildEdition` already encodes that information, and it looks like `$User` meant to do that but got superseded and left in